frontend/public/index.html

```diff
@@ -6,11 +6,11 @@
 
     <title>Svelte app</title>
 
-    <link rel="icon" type="image/png" href="/favicon.png" />
-    <link rel="stylesheet" href="/global.css" />
-    <link rel="stylesheet" href="/build/bundle.css" />
+    <link rel="icon" type="image/png" href="favicon.png" />
+    <link rel="stylesheet" href="global.css" />
+    <link rel="stylesheet" href="bundle.css" />
```

Comment:
Why did you move the files out of the build directory?

Reply:
oh thanks for looking, couldn't even ask for a review ;)

I started this work as frontend cleanup/initial PR.
I did it because I think I kinda like things un-nested if there are that few. 
I know this mixes generated with manually created content, but I thought the tradeoff is ok, compared to the extra logic I had to add at other places to move the result files into one directory...